### PR TITLE
Added java 13 package in VM

### PR DESCRIPTION
**Pull request by sima-zhu**
# Description
New tool, Bug fixing, or Improvement?  
Please include a summary of the change and which issue is fixed. Also include relevant motivation and context.  
**For new tools, please provide total size and installation time.**
Download and Install Java 13 to VM.
**Total size**: 199880KB
**Installation time**: Not quite sure. Please let me know how I can get the time.
#### Related issue:
https://github.com/Azure/azure-sdk-for-java/issues/10465
## Check list
- [x] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/win/scripts/Installers/Install-JavaTools.ps1 [hunk 1]
```diff
@@ -49,6 +53,9 @@ $latestJava8Install = $java8Installs.FullName;
 $java11Installs = Get-ChildItem -Path 'C:\Program Files\Java' -Filter '*azure-jdk*11*' | Sort-Object -Property Name -Descending | Select-Object -First 1
 $latestJava11Install = $java11Installs.FullName;
 
+$java11Installs = Get-ChildItem -Path 'C:\Program Files\Java' -Filter '*azure-jdk*13*' | Sort-Object -Property Name -Descending | Select-Object -First 1
+$latestJava11Install = $java11Installs.FullName;
```
Comment (maxim-lobanov) [inline images/win/scripts/Installers/Install-JavaTools.ps1:57]: I think it should be different variable `latestJava13Install`

**Comment by maxim-lobanov** [comment]
LGTM, I will start CI builds

**Comment by sima-zhu** [comment]
Thanks for moving this forward!
@maxim-lobanov The build has passed.  What is the next step? How can we use the Java 13 in devop?

**Comment by maxim-lobanov** [comment]
@sima-zhu , as soon as PR is merged, it will take about 1 week to propagate these changes to all VMs

@alepauly , any concerns about merging this one?

**Comment by sima-zhu** [comment]
Any updates on this? We have features need to test on Java 13. This is a blocker for releasing the feature. It would be great if we can this change checkin this week.
Thanks!

**Comment by alepauly** [comment]
@sima-zhu @maxim-lobanov Sorry, I had missed this. I believe we need some changes to validate and document the version in Validate-JavaTools.ps1, is that correct?

**Comment by alepauly** [comment]
@sima-zhu if missing Java 13 is blocking for you, I believe you should be able to install during your builds in the meantime.

**Comment by sima-zhu** [comment]
@alepauly Could you explain a little how to install Java 13 during build? Do you mean I manually added this file into our yml file?

**Comment by maxim-lobanov** [comment]
> @alepauly Could you explain a little how to install Java 13 during build? Do you mean I manually added this file into our yml file?

I think that something like that will work for you:
```
$azulJDK13Uri = 'https://repos.azul.com/azure-only/zulu/packages/zulu-13/13.0.3/zulu-13-azure-jdk_13.31.11-13.0.3-win_x64.zip'
Invoke-WebRequest -UseBasicParsing -Uri $azulJDK13Uri -OutFile azulJDK13.zip
Expand-Archive -Path azulJDK13.zip -DestinationPath "C:\Program Files\Java\" -Force
$java13Installs = Get-ChildItem -Path 'C:\Program Files\Java' -Filter '*azure-jdk*13*' | Sort-Object -Property Name -Descending | Select-Object -First 1
$latestJava13Install = $java13Installs.FullName;
setx JAVA_HOME_13_X64 $latestJava13Install /M
```

We will be able to merge this PR right after fixing comment ` I believe we need some changes to validate and document the version in Validate-JavaTools.ps1, is that correct?`
But please pay attention that propagation of these changes to all VMs will take 1-2 weeks so it blocks your release, I suggest to install it in runtime as a workaround

**Comment by sima-zhu** [comment]
I have updated the `Validate-JavaTools`
Could you take a look @maxim-lobanov ?

**Comment by sima-zhu** [comment]
@maxim-lobanov Thanks for taking care of this! 
Can we merge the code?

**Pull request merged**